tools/JsonCli/src/Microsoft.TemplateEngine.JsonCli.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFrameworks>netcoreapp2.0;netcoreapp3.0;net5.0;net6.0</TargetFrameworks>
+        <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
There's several other places you'd need to update. I think for this PR, this file might be the only place:
https://github.com/microsoft/vscode-azurefunctions/blob/main/tools/JsonCli/.azure-pipelines/main.yml

After all that gets merged/built you'll need to update the bits here:
https://github.com/microsoft/vscode-azurefunctions/tree/main/resources/dotnetJsonCli

And the logic here:
https://github.com/microsoft/vscode-azurefunctions/blob/main/src/templates/dotnet/executeDotnetTemplateCommand.ts#L47

Reply:
Good to know, thanks @ejizba!

Reply:
Updated the signing project too.

Reply:
Followup PR is #3018.